`/var/lib/jenkins/jobs/dron/workspace@3/Nodes/master/build/bu_brainre1/bu_brainre1.bin.firmwareinfo.c:75:24: error: invalid suffix "xNone" on integer constant
  .next_ancestor_hash = 0xNone,
                        ^
make[1]: *** [/var/lib/jenkins/jobs/dron/workspace@3/Nodes/master/build/bu_brainre1/bu_brainre1.bin.firmwareinfo.o] Error 1
make: *** [bu_brainre1_tlfw] Error 2`

Don't get this on my machines?

--- a/package/Makefile
+++ b/package/Makefile
@@ -136,6 +136,6 @@ ground_package_common: standalone
 package_flight: $(FLIGHTPKGNAME)
 
 $(FLIGHTPKGNAME): $(PACKAGE_DIR) $(FW_FILES)
-	$(ZIPBIN) -j $@ $(FW_FILES)
+	$(ZIPBIN) -j --must-match $@ $(FW_FILES)
 
 include $(WHEREAMI)/Makefile.$(PLATFORM)

--- a/package/Makefile
+++ b/package/Makefile
@@ -135,7 +135,7 @@ ground_package_common: standalone
 .PHONY: package_flight
 package_flight: $(FLIGHTPKGNAME)
 
-$(FLIGHTPKGNAME): $(PACKAGE_DIR)
+$(FLIGHTPKGNAME): $(PACKAGE_DIR) $(FW_FILES)
 	$(ZIPBIN) -j $@ $(FW_FILES)
 
 include $(WHEREAMI)/Makefile.$(PLATFORM)

